packages/nest/src/generators/convert-tslint-to-eslint/schema.json

```diff
@@ -21,6 +21,11 @@
       },
       "x-prompt": "Which NestJS project would you like to convert from TSLint to ESLint?"
     },
+    "discardExistingLintConfig": {
```

Comment:
Can we please rename this to `ignoreExistingTslintConfig`?
  1. We're not discarding anything (unless tslint is entirely gone)
  2. This specifically ignores tslint config
  3. This shouldn't overwrite existing eslint config right?

Reply:
Renamed